### ISSUE TITLE
Manual Deploy Script could use work

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/edu-api-front-end#34]
## Investigation Needed
This follows PR #33 and its extensive _list of known WTFs_

Issue 1:  What's wrong with ExtractTextPlugin in webpack.production.config.js?
* Original code:
```
test: /\.css$/,
use: ExtractTextPlugin.extract({
  fallback: 'style-loader',
  loader: 'css-loader',
  options: {
    includePaths: [
      path.resolve(__dirname, 'node_modules/zoo-grommet/dist'),
      path.resolve(__dirname, 'node_modules/zooniverse-react-components/lib')
    ]
  }
}),
```
* PR 33 removed `ExtractTextPlugin` to dodge an error. Is there a better way to solve this?

Issue 2: 💀  Why does the built & deployed website ( https://classroom.zooniverse.org ) thinks it's on a staging/development environment?

Issue 3: Why does the script for `npm run deploy-production` have to be so unnecessarily long and duplicate the script for `npm run build`?

Issue 4: Is there a better alternative to `publisssh`?

### Status
Open season for investigation. Only Issue 2 is considered a high priority item since it affects our end users.

Tagging @srallen into this; Sarah, I'd appreciate your insights into this when you're back.

**Comment by srallen** [comment]
ExtractTextPlugin is used to extract the CSS into a css file rather than the default webpack behavior which is to inject it into a style tag in the head of the HTML. There's likely something else wrong, perhaps the configuration rather than the plugin itself. We'll want the plugin to work, so the main.css file builds correctly.

If I get a chance, I'll take a look at your other questions.

**Comment by srallen** [comment]
Ok, ignore what I'm saying partly. The stylus configuration is handling the extraction, so it probably is correct to remove the plugin from the css loader configuration. I'll double check to confirm.

**Comment by srallen** [comment]
Closed by #36
